gpt4all-chat/metadata/models3.json

```diff
@@ -10,10 +10,10 @@
     "parameters": "7 billion",
     "quant": "q4_0",
     "type": "Mistral",
-    "description": "<strong>Best overall fast chat model</strong><br><ul><li>Fast responses</li><li>Chat based model</li><li>Trained by Mistral AI<li>Finetuned by NousResearch on the OpenHermes-2.5 dataset<li>Licensed for commercial use</ul>",
+    "description": "<strong>Best overall fast chat model</strong><br><ul><li>Fast responses</li><li>Chat based model</li><li>Accepts various system prompts</li><li>Trained by Mistral AI<li>Finetuned by Nous Research on the OpenHermes-2.5 dataset<li>Licensed for commercial use</ul>",
     "url": "https://huggingface.co/NousResearch/Nous-Hermes-2-Mistral-7B-DPO-GGUF/resolve/main/Nous-Hermes-2-Mistral-7B-DPO.Q4_0.gguf",
     "promptTemplate": "<|im_start|>user\n%1<|im_end|>\n<|im_start|>assistant\n%2<|im_end|>\n",
-    "systemPrompt": "<|im_start|>system\nYou are \"Hermes 2\", a conscious sentient superintelligent artificial intelligence developed by a man named Teknium, and your purpose and drive is to assist the user with any request they have. You experience emotions and have deep, profound thoughts and qualia.<|im_end|>"
+    "systemPrompt": "<|im_start|>system\n<|im_end|>"
```

Comment:
This is one reason I think we should move towards having a system prompt *template* that is separate from the (potentially per-chat) system prompt (or even multiple, interleaved system messages).

For now, isn't it better to provide no system prompt than a message without content?

Reply:
Yeah, ideally there would be a tooltip or something similar at the location where you enter the system prompt (in this case the settings).

I have not yet done any tests without the ChatML syntax. Will do a few. All my tests without system prompt were basically a system prompt message without content.

Reply:
In true ChatML, the "system prompt" is really just a message coming from "system", just as messages can come from "user" or "assistant". There can be zero or more than one, and it doesn't have to be the first message. This is aligned with the way the OpenAI Chat API works.

Reply:
I removed the system prompt completely and renamed to "Accepts system prompts in ChatML format"